quick fixes (#8)
LIT-Protocol/vincentDocs: docs/Users/Onboarding.md

# Onboarding

> Each time you visit an app on Vincent, you’ll be directed to that app’s dedicated consent page. This is where you will configure and manage your app-specific delegations, permitting it to use your assets and data according to the guidelines you set. Follow the steps below to begin your journey

Reviewer: ```suggestion
Each time you visit an app on Vincent, you’ll be directed to that app’s dedicated consent page. This is where you will configure and manage your app-specific delegations, permitting it to use your assets and data according to the guidelines you set. Follow the steps below to begin your journey.
```

## 1. Create an Agent Wallet

> The first time you visit an app powered by Vincent, you’ll be prompted to create your Agent Wallet using a credential of your choice. Currently, email, phone number, and Passkey methods are supported. This credential is used to control your Agent Wallet which is securely managed by Lit Protocol’s [decentralized key management network](https://developer.litprotocol.com/resources/how-it-works). You will have the ability to use the same Agent Wallet across all Vincent apps, or create a new one if you choose to.

Reviewer: > or create a new one if you choose to.

I don't think we support generating a new wallet using the same Auth Method, FYI. The preference was for a user to only ever have a single Agent Wallet used for every Vincent App

After you login you'll be required to review the permissions requested by the app on the Consent Page. On this page you can see the following details about the app:

    - **App:** The name and description of the app you're visiting.
    - **Account Address:** The EOA wallet address associated with the App creator.
    - **App Version:** The version of the app you're visiting. If the app publishes a new version you'll be prompted to update it.

>     - **Tools:** Tools are used to execute the operations that you permit a given app to perform on your *behalf* using your Agent Wallet. All Tools are represented by [Lit Actions](https://developer.litprotocol.com/sdk/serverless-signing/overview). After you delegate your Agent Wallet to a given Tool, the application owner can no longer change the Tool's behavior, making the delegation system trustless. As the owner of your Agent Wallet, you always have the ability to revoke a Tool that you previously permitted. 

Reviewer: ```suggestion
    - **Tools:** Tools are used to execute the operations that you permit a given app to perform on your *behalf* using your Agent Wallet. All Tools are represented by [Lit Actions](https://developer.litprotocol.com/sdk/serverless-signing/overview). After you permit a Tool to be used with your Agent Wallet for a specific app, the app owner can no longer change the Tool's behavior, making the delegation system trustless. As the owner of your Agent Wallet, you always have the ability to revoke a Tool that you previously permitted for any Vincent app. 
```

> making the delegation system trustless.

It's also not technically trustless, it's more appropriate to say it's immutable, but trustless does sound better
